src/test/tasks.yaml

```diff
@@ -148,6 +148,28 @@ tasks:
             namespace: podinfo
             condition: "'{.status.phase}'=Ready"
 
+      - description: Verify podinfo namespace has ambient mode label
+        cmd: |
+          set -e
+          AMBIENT_LABEL=$(uds zarf tools kubectl get namespace podinfo -o jsonpath='{.metadata.labels.istio\.io/dataplane-mode}')
```

Comment:
Would it be beneficial to move these tests to `jest`?

Reply:
Migrated these to jest on the latest commit.